### PR TITLE
Cleanup of documentation, linchpin_config.yml

### PR DESCRIPTION
**Pull request by herlo**
Update to README.md with values now being used
Modified linchpin_config.yml to remove reference to inventory_layout_file
Delete output and inventory directories before recreating them to ensure files don't get reused.

### CODE REVIEW

File: provision/hosts [hunk 1]
```diff
@@ -1,2 +0,0 @@
-localhost ansible_connection=local
```
Comment (arilivigni) [inline provision/hosts]: Are you sure this isn't needed any longer?

Comment (herlo) [inline provision/hosts]: It works without this value on slave01. I suspect we could leave it, however. Let me update that in my commits.

**Comment by herlo** [comment]
Replaced the provision/hosts file as requested. Merging with comment from @arilivigni in irc

15:06 <@ari> herlo: I only make that comment because it was added back to support the way we provision but maybe it is no longer needed
15:07 < herlo> ari: yeah. I don't know if it will cause problems elsewhere. You made a good point about it.
15:07 <@ari> other than that LGTM

File: README.md [hunk 2]
```diff
@@ -161,20 +188,71 @@ For GCE the absolute path of the service account json file should be provided
...
 
-### De-Provision a topology
 ```
-command: ansible-playbook -vvv provision/site.yml -e "{'data':'path_to_topolgy_file', 'schema':'path_to_schema_file', 'state':'absent', 'topology_output_file':'path_to_output_file'}"
+## CLI Examples
+### Provision a topology
 
-example: ansible-playbook -vvv provision/site.yml -e "{'data':'../ex_topo/ex_data.yml', 'schema':'../ex_schemas/schema_v2.json', 'state':'absent', 'topology_output_file':'/tmp/ex_data_output.yaml'}"
 ```
+$ ansible-playbook -vvv provision/site.yml -e "topology='/path/to/topology_file'" \
```
Comment (samvarankashyap) [inline README.md]: @herlo  : Why are there multiple -e args instead of single -e with multiple variables inside it ?

Comment (greg-hellings) [inline README.md]: @samvarankashyap This syntax is simpler for people to read an understand than trying to write and escape JSON on the command line.

Comment (herlo) [inline README.md]: For documentation, it's clearer to understand from a user perspective. I've been doing a bit of cleaning up to make things more simplistic to use. I'm hoping the update to the documentation has helped others understand how the provisioning works.

**Comment by dygrys** [comment]
Hi I would like to create masurian version of Firefox, how can I do it?

**Pull request merged**